test/chi/components/header.pug

```diff
@@ -6,40 +6,53 @@ h3 Base
 .test-header-base.-w--100
```

Comment:
@sabah1910 please review mobile versions in base examples as they are not rendering the same as in doc

Reply:
@jllr  What is meant by Mobile Versions? I am looking at examples at https://assets.ctl.io/chi/2.1.0/components/header/, is it different?

Reply:
@sabah1910 by mobile versions I mean you must resize the browser in to see it correctly.